components/Map.js

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import ReactMapGL, { Marker } from "react-map-gl";
+export default function Map({ locations }) {
+  // UseState hook
+  const [selectedLocation, setSelectedLocation] = useState({});
```

Comment:
Taking in mind the main users of the platform should be in Cuenca, why not to set as default location, Cuenca location.

Reply:
I have changed the useState in the map page. I don't understand very good this library

Reply:
I have changed the useState in the map page. I don't understand very good this library